core/src/test/scala/unit/kafka/server/ApiVersionsRequestTest.scala

```diff
@@ -44,6 +44,13 @@ class ApiVersionsRequestTest(cluster: ClusterInstance) extends AbstractApiVersio
     validateApiVersionsResponse(apiVersionsResponse)
   }
 
+  @ClusterTest(serverProperties = Array(new ClusterConfigProperty(key = "unstable.api.versions.enable", value = "true")))
```

Comment:
Just curious: do we need the serverProperties given that we call the validate func below with specific `enableUnstableLastVersion` set already?

Reply:
Yes, we need both. `enableUnstableLastVersion = true` is to actually validate that the unstable is correctly exposed.